src/transformers/models/whisper/tokenization_whisper.py

```diff
@@ -988,3 +992,12 @@ def _find_longest_common_sequence(sequences):
     total_sequence.extend(left_sequence)
 
     return total_sequence
+
+
+def _strip_prompt(token_ids: List[int], prompt_token_id: int, all_special_ids: List[int]):
```

Comment:
Can also make this a static method and move inside the class `WhisperTokenizer`:
```python
@staticmethod
def _strip_prompt(token_ids: List[int], prompt_token_id: int, all_special_ids: List[int]):
```